metadata.py

```diff
@@ -40,15 +40,26 @@ def appmetadata() -> AppMetadata:
         # analyzer_license="",  # short name for a software license
     )
     # and then add I/O specifications: an app must have at least one input and one output
-    metadata.add_input(AnnotationTypes.TimeFrame)
+    metadata.add_input(AnnotationTypes.TimeFrame)   # Output from SWT
+    metadata.add_input(DocumentTypes.TextDocument)  # Output from OCR
     metadata.add_output(DocumentTypes.TextDocument)
+    metadata.add_output(AnnotationTypes.Alignment)
     
     # (optional) and finally add runtime parameter specifications
-    metadata.add_parameter(name='a_param', description='example parameter description',
-                           type='boolean', default='false')
-    # metadta.add_parameter(more...)
-    
-    # CHANGE this line and make sure return the compiled `metadata` instance
+    metadata.add_parameter(name='containLabel',
+                           description=
+                           '''
```

Comment:
this will end up like this 
``` json 
 $ curl localhost:5000 | jq 
{
  ...
  "parameters": [
    {
      "name": "containLabel",
      "description": "\n                           A list of labels that user expect TimeFrames contain.\n                           Available options are:\n                           ['bars', 'tones', 'bars-and-tones','speech',\n                           'noise', 'music', 'slate', 'chyron',\n                           'lower-third', 'credits']\n                           Users must select at least one of the following labels, errors would \n                           be thrown instead.\n                           ",
      ...
    },
    ...
  ]
  ...
}
```
When using three-quotes string, be aware of the difference btw beautiful code and actual output. 

Here, I think what you really want is not a "multi-line" string ([pep-295](https://peps.python.org/pep-0295/), rejected) but a string value in continuation lines ([pep-8](https://peps.python.org/pep-0008/), also [string literal in python parser reference](https://docs.python.org/3/reference/lexical_analysis.html#string-literal-concatenation).

Reply:
Also, "Available options" are really limited to those items, so it'd better to say that the list is "non-exhaustive", or "not limited to", or change the wording from "Available options" to "For example" or something like that.

Reply:
>When using three-quotes string, be aware of the difference btw beautiful code and actual output.

I agree. Since I tested the app description in CLI by calling `python cli.py -h`, but the description hasn't beed divided into multi-lines if I add `\n` in the string. 

Now I changed the string format of this argument as 
```
 add_parameter(name=....
               description=".....\n"
                           ".....\n"
                           ".....\n"
               other_params=...
               )
```
So that it makes both code and actual output pretty. What's your thought?